DESCRIPTION

```diff
@@ -56,6 +56,7 @@ Suggests:
     knitr,
     magick,
     mockr,
+    pkgdown,
```

Comment:
I suspect this will prove to be unnecessary.

Reply:
I added it because something was failing, but now I don't see what. Maybe I removed that code?

Reply:
I think the failure may have had another explanation. I'll handle this.